src/main/java/com/cburch/logisim/fpga/hdlgenerator/RGBArrayRowScanningHDLGeneratorFactory.java

```diff
@@ -147,7 +147,8 @@
           "",
           "genvar i;",
           "generate",
-          "   for (i = 0; i < {{nrOfColumns}}; i = i + 1) begin",
+          "   for (i = 0; i < {{nrOfColumns}}; i = i + 1)",
+          "   begin:outputs",
           "      assign {{outsR}}[i] = (activeLow == 1)",
           "         ? ~{{insR}}[{{nrOfColumns}} * s_rowCounterReg + i]",
           "         : {{insR}}[nrOfColumns * s_rowCounterReg + i];",
```

Comment:
Is the last line correct:

```java
                assign {{outsR}}[i] = (activeLow == 1)
                   ? ~{{insR}}[{{nrOfColumns}} * s_rowCounterReg + i]
                   :  {{insR}}[nrOfColumns * s_rowCounterReg + i];
```

For me it looked like we should have `{{nrOfColumns}}` instad of `nrOfColumns`, so I corrected this.